From 606843ad079d9d0dac1172533d42b5a2327b99ed Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 17 Sep 2025 15:41:09 +0800
Subject: [PATCH] build wheel in place

In order to make the generated library be reproducible, build wheel
in source dir other than tmp dir (/tmp/xxxxxx), then yocto toolchain's
option -fdebug-prefix-map could work as expected

Upstream-Status: Pending
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 packaging/pep517_backend/_backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/pep517_backend/_backend.py b/packaging/pep517_backend/_backend.py
index 61cad34..d5d839e 100644
--- a/packaging/pep517_backend/_backend.py
+++ b/packaging/pep517_backend/_backend.py
@@ -316,7 +316,7 @@ def build_wheel(
     """
     with maybe_prebuild_c_extensions(
             line_trace_cython_when_unset=False,
-            build_inplace=False,
+            build_inplace=True,
             config_settings=config_settings,
     ):
         return _setuptools_build_wheel(
-- 
2.34.1

